template/console/serverless.yml

```
@@ -12,6 +12,7 @@ functions:
     console:
         handler: bin/console # Replace by `artisan` if you are using Laravel
         description: ''
+        timeout: 30 # in seconds (API Gateway has a timeout of 30 seconds)
```

Since we are running a console command, we aren't using API Gateway. But 30 seconds is a good timeout anyway, we just need to edit the comment, for example:

```suggestion
        timeout: 30 # in seconds
```